lib/engines/content_block_manager/features/reschedule_object.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Schedule a content object
+  Background:
+    Given I am a GDS admin
```

Comment:
this move might be a bit controversial but thought it worth getting some feedback on